core/domain/exp_jobs_one_off.py

```diff
@@ -503,13 +506,20 @@ def map(item):
         for html_data in html_list:
             soup = bs4.BeautifulSoup(html_data, 'html.parser')
 
+            # Any html string with unwrapped text is also invalid.
+            for content in soup.contents:
+                if not content.name:
+                    err_dict['strings'] += [html_data]
```

Comment:
For all these it's better to use .append(). It makes it clearer that you're appending an item to a list, which really is what you're doing.

Reply:
done!